cmd/podman/parse/common.go

```diff
@@ -30,13 +30,17 @@ func CheckAllLatestAndCIDFile(c *cobra.Command, args []string, ignoreArgLen bool
 		return errors.Errorf("--all and --latest cannot be used together")
 	}
 
+	if (argLen > 0) && specifiedAll {
+		return errors.Errorf("no arguments are needed with --all")
```

Comment:
Changes this to state
"ports are not allowed with the --all option"

Reply:
This is a common check and not a specific one.